console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/setting/ConsoleSettingKeys.java

```diff
@@ -39,6 +39,7 @@
 
     SSO_REDIRECT_URI("console.sso.redirect.uri"), //
     SSO_CONSOLE_HOME_URI("console.sso.home.uri"), //
+    SSO_OPENID_LOGOUT_ENABLED("console.sso.openid.logout.enabled"), //
```

Comment:
Can you please change the header to "2020"? I hope I am not too late 😄

Reply:
done!